backend.go

```diff
@@ -2,13 +2,16 @@ package jwtauth
 
 import (
 	"context"
+	"errors"
 	"sync"
 	"time"
 
-	oidc "github.com/coreos/go-oidc"
+	"github.com/hashicorp/errwrap"
```

Comment:
Probably should be moved down to go with the other hashicorp imports.

Reply:
Yep